pkg/instance/instance.go

```diff
@@ -1,12 +1,33 @@
 package instance
 
 import (
+	"fmt"
+	"net"
+
+	"github.com/pkg/errors"
 	core "k8s.io/api/core/v1"
 
 	"github.com/openshift/windows-machine-config-operator/pkg/metadata"
 	"github.com/openshift/windows-machine-config-operator/version"
 )
 
+// NoIPv4Error occurs when trying to retrieve an IPv4 address from an instance whose network address is IPv6 or a DNS
+// name that does not resolve to an IPv4 address
+type NoIPv4Error struct {
```

Comment:
Do we really need the `Error` suffix?

Reply:
I see k8s errors do not have the `Error` suffix so I will remove follow that convention.